libvast/src/system/index.cpp

```diff
@@ -231,10 +212,28 @@ partition_ptr index_state::make_partition(uuid id) {
   return std::make_unique<partition>(this, std::move(id), max_partition_size);
 }
 
-caf::actor index_state::make_indexer(path dir, type column_type,
-                                     size_t column) {
-  VAST_TRACE(VAST_ARG(dir), VAST_ARG(column_type), VAST_ARG(column));
-  return factory(self, std::move(dir), std::move(column_type), column);
+caf::actor index_state::make_indexer(path dir, type column_type, size_t column,
+                                     uuid pid) {
```

Comment:
So far, we've never used `pid` to refer to a partition ID. Since this is a very common abbreviation for *process ID*, I think we should spend the extra characters and consistently use `partition_id` as variable name.